svgo.config.js

```diff
@@ -0,0 +1,124 @@
+const { extendDefaultPlugins } = require('svgo');
+
+module.exports = {
+  multipass: true,
+  plugins: extendDefaultPlugins([
+    {
+      name: 'convertPathData',
+      params: {
+        // 3 decimals of precision in floating point numbers
+        floatPrecision: 3,
+        // Some editors (eg. Adobe Illustrator and Sketch) does not parse flags
+        // without space wrapping
```

Comment:
Minor grammatical mistake, but still very much understandable

```suggestion
        // Some editors (e.g. Adobe Illustrator and Sketch) cannot parse flags
        // without space wrapping
```